.github/workflows/build-doxygen.yml

```diff
@@ -21,17 +27,78 @@ jobs:
 
     steps:
     - name: Get repository
-      uses: actions/checkout@v5
+      uses: actions/checkout@v4
```

Comment:
Can you please fix all of the calls to actions/checkout to version 5. Thank you.